### PR TITLE
Reslug News Frequently Asked Questions

### PR DESCRIPTION
**Pull request by mgrassotti**
Trello: https://trello.com/c/YYaiCU7Y/548-slug-change-required

Modify the slug of the document into
"Information-about-the-UK-leaving-the-EU"

### CODE REVIEW

File: db/data_migration/20161222102639_reslug_news_faq.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+article = NewsArticle.find(688810)
+document = article.document
+
+document.update_attributes(slug: "Information-about-the-UK-leaving-the-EU")
```
Comment (fofr) [inline db/data_migration/20161222102639_reslug_news_faq.rb]: Should it be `information` not `Information`?

Comment (fofr) [inline db/data_migration/20161222102639_reslug_news_faq.rb]: Zendesk ticket has been updated saying it should be lowercase: https://govuk.zendesk.com/agent/tickets/1480095

**Comment by mgrassotti** [comment]
👍

**Comment by fofr** [comment]
@boffbowsh It is now, isn't it? That's why I approved?

**Comment by boffbowsh** [comment]
I force-pushed the fix after I commented, once I remembered Matteo was off all week

**Pull request merged**